### PR TITLE
Add vagrant-option to use rsync instead of nfs

### PR DESCRIPTION
**Pull request by cashewkernchen**
See https://docs.vagrantup.com/v2/synced-folders/rsync.html for rsync options.

### CODE REVIEW

**Comment by bitboxer** [comment]
Can you add a hint to that to the readme in the vagrant section?

File: Vagrantfile [hunk 1]
```diff
@@ -26,6 +26,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.hostname = 'hacken'
 
   config.vm.synced_folder ".", "/opt/hacken.in", type: 'nfs'
+  # rsync fallback if NFS has issues issues with encrypted filesystems
```
Comment (thegcat) [inline Vagrantfile]: issues issues :-)

**Comment by bitboxer** [comment]
But other than that: works on my machine. Thanks for this :heart:

**Comment by bitboxer** [comment]
Thanks! Merging this now.

**Pull request merged**